### ISSUE TITLE
copyExternalStyle of rappid.js not correctly working on Firefox  : CSS2Properties doesn't have an indexed property setter for '15' error 

### ISSUE THREAD

**Issue by koenhandekyn** [clientIO/joint#848]
when saving PNG/SVG on firefox > error in log : CSS2Properties doesn't have an indexed property setter for '15' error.

analysis: 

the root cause comes from function copyExternalStyle of rappid.js / jointjs.  we have 

```
computedStyle = window.getComputedStyle(el,null)

computedStyle = {
// key, property
0: "align-content"
1: "align-items"
2: "align-self"
3: "animation-delay"
...
'align-content': 'normal',
'align-items': 'normal',
'stroke-width': 15,
}

```
Then, this line is the root cause of this bug

```
customStyle[property] = computedStyle.getPropertyValue(property), eg. property = "align-content" ,
=> getPropertyValue("align-content") = "normal"

```
customStyle will be:

```
customStyle= {
 align-content: 'normal',
 align-items: 'normal',
15: ""
}

```
the key "15": cause bug, in the last line when we set $(el).css(customStyle[idx])

**

> error in log : CSS2Properties doesn't have an indexed property setter for '15' error

**

it doesn't happen in chrome: I tested and found that when setting CSS in chrome by jquery key like "15:" will not be set and don't raise any errors

Solution:

When set customStyle[property] = computedStyle.getPropertyValue(property). we only set the property with key is number, 0,1,2,3,4... . because some keys like "15" will not be set to CSS in the end.

**Comment by koenhandekyn** [comment]
can you highlight in the above commit where you changed what ?

**Comment by koenhandekyn** [comment]
this might be relevant as an alternative solution

https://stackoverflow.com/questions/46077237/how-to-solve-css2properties-doesnt-have-an-indexed-property-setter-for-0-erro

**Comment by kumilingus** [comment]
Fixed in Rappid `v2.3.1`.
